.github/workflows/centraldb_angular_integration_test.yaml

```diff
@@ -13,8 +13,8 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  IMG: centraldashboard-angular
-  TAG: intergration-test
+  IMG: ghcr.io/kubeflow/kubeflow/centraldashboard-angular
```

Comment:
Thank you for doing this @ederign!
@kubeflow/wg-manifests-leads @kubeflow/wg-notebooks-leads @ederign Given that we are in the process of migrating the repositories to the `kubeflow/dashboard` and `kubeflow/notebooks`, what do you think about using the new location for the 1.9.0 release ?
```
ghcr.io/kubeflow/dashboard/central-dashboard:v1.9.0-SHA

ghcr.io/kubeflow/notebooks/notebook-controller:v1.9.0-SHA
```


Since in any case we've been using this image: `docker.io/kubeflownotebookswg` for the 1.4 ... 1.9 releases: https://github.com/orgs/kubeflow/packages/container/package/kubeflow%2Fcentral-dashboard


I feel like if we configure the actions in this PR to use the newer location, the migration will be easier after it.
cc @milinddethe15

Reply:
@andreyvelich I agree that this is the right time to do this migration. My only word of warning is that we need to converge to a decision quickly, because my understanding is this blocks 1.10 release.